docs/docs/ref-01-top-level-api.md

```diff
@@ -139,7 +139,7 @@ The `react-dom` package provides DOM-specific methods that can be used at the to
 ### ReactDOM.render
 
 ```javascript
-render(
+void render(
```

Comment:
Hmm, the issue here is that the return type isn't technically void (though it would be good if people treated it that way).